src/town.h

```diff
@@ -101,6 +101,8 @@ struct Town : TownPool::PoolItem<&_town_pool> {
 	bool larger_town;              ///< if this is a larger town and should grow more quickly
 	TownLayoutByte layout;         ///< town specific road layout
 
+	bool show_zone;                ///< mark town to show the local authority zone in the viewports
```

Comment:
should have a NOSAVE label, i think

Reply:
Yes, this data should not be saved. Is the label enough, or I also have to set is elsewhere?